src/main.cpp

```diff
@@ -3930,8 +3930,10 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         return true;
     }
 
-    // DS: update last valid block from node
+    {
```

Comment:
nLastBlockProcess has been removed so this changed should be reversed to remove the conflict